### DISCUSSION TITLE
Using icons

### DISCUSSION THREAD

**Discussion by accessburn** [cgeo/cgeo#13801] [Show and tell]
Hi there :-)

Lineflyer said to write here again too.

@pstorch and I are tinkering together on a small event app (github.com/accessburn/GcffmApp) and want to use the some icons from you. May we use the "type-icon.xmls" from the event types?

Unfortunately I can only offer you the info in the about and the readme in return.

Greetings from Ffm,
Toby

**Comment by ztNFny** [comment]
ok with me. In case you need something other than what's available in c:geo, the svg sources used are in https://github.com/cgeo/cgeo/tree/master/main/project/design/cacheicons

**Comment by Lineflyer** [comment]
Fine for me as well

**Comment by SammysHP** [comment]
Wow, that app is still alive? Nice!

Regarding icons: Fine for me.
